### PR TITLE
 Implementation Edit to Bubble Sort in Javascript

### PR DESCRIPTION
**Pull request by CDsigma**
Added main() to code so that it can be run as is

### CODE REVIEW

File: contents/bubble_sort/code/javascript/bubble.js [hunk 1]
```diff
@@ -8,4 +8,12 @@ function bubbleSort(arr) {
       }
     }
   }
+  return arr;
```
Comment (Butt4cak3) [inline contents/bubble_sort/code/javascript/bubble.js]: Same thing as in the bogo sort edit. You don't actually have to return the array.

File: contents/bubble_sort/code/javascript/bubble.js [hunk 2]
```diff
@@ -8,4 +8,12 @@ function bubbleSort(arr) {
...
 }
+
+function main() {
+	var testArray = [4,5,123,759,-132,8940,24,34,-5]
+	print(bubbleSort(testArray));
```
Comment (Butt4cak3) [inline contents/bubble_sort/code/javascript/bubble.js]: No `print` in JavaScript. Use `console.log` and split it up into two lines.

```JavaScript
bubbleSort(testArray);
console.log(testArray);
```

File: contents/bubble_sort/code/javascript/bubble.js [hunk 3]
```diff
@@ -8,4 +8,12 @@ function bubbleSort(arr) {
       }
     }
   }
+  return arr;
 }
+
+function main() {
+	var testArray = [4,5,123,759,-132,8940,24,34,-5]
```
[the remark this answers is not in the document]
Comment (Butt4cak3) [reply contents/bubble_sort/code/javascript/bubble.js]: Wow. I didn't catch that. GitHub renders the indentation inside `main` two spaces wide.

**Comment by julianschacher** [comment]
Same with this one, changes seemed to have happened. How's it looking? @Butt4cak3 @Gustorn

**Comment by Gustorn** [comment]
Yeah, looks good to me. @Butt4cak3 we need you to approve it

**Pull request merged**